### PR TITLE
ci: install Yarn properly

### PR DESCRIPTION
**Pull request by DanielRuf**
### Please check if the PR fulfills these requirements ###

- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
Bug fix and docs update.

**What is the current behavior? (You can also link to an open issue here)**
Yarn is installed with npm, which should not be the case.

**What is the new behavior (if this is a feature change)?**
Yarn is installed according to the official docs.

**Does this PR introduce a breaking change? (What changes might users need to make in their application due to this PR?)**
No.

**Other information:**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,7 +5,8 @@ node_js:
 cache: yarn
 
 before_install:
-  - npm i -g yarn
+  - curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.10.1
```
Comment (Borales) [inline .travis.yml]: @DanielRuf I'm ok with your solution, but I don't think we need a specific version here. Maybe you have a specific concern about this?

Comment (DanielRuf) [reply .travis.yml]: This is from https://yarnpkg.com/lang/en/docs/install-ci/

Comment (DanielRuf) [reply .travis.yml]: Without the version flag it would install the latest release at this time.

Comment (Borales) [reply .travis.yml]: @DanielRuf yes, good point. Thank you!

**Pull request merged**